pkgs/desktops/gnome/artwork/firewatch/default.nix

```diff
@@ -0,0 +1,230 @@
+{ stdenv
+, lib
+, fetchzip
+, gnome
+}:
+
+stdenv.mkDerivation rec {
+  pname = "wallpaper-firewatch";
+<<<<<<< HEAD
+  version = "2.1";
+=======
+  version = "0.1";
+>>>>>>> a11990b425c31bde2882fa77d926db610b5b563f
+  src = fetchzip {
+    url = "https://cdn.manishk.dev/Firewatch.zip";
+    sha256 = "sha256-DSG4oGOJDXFKatCL43bO93++y4pxg1Ygxt9kC3v+HvM=";
+    stripRoot = false;
+  };
+
+  installPhase = ''
+        runHook preInstall
+        mkdir -p $out/share/backgrounds/animated/Firewatch
+        mv *.jpg $out/share/backgrounds/animated/Firewatch
+        cat <<EOF > $out/share/backgrounds/animated/Firewatch.xml
```

Comment:
Just use the XML files from the tarball and fix the path in `postPatch` with `substituteInPlace`.